# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hpgl-distiller/cross-cc.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hpgl-distiller-0.9.1/Makefile.vanilla	2007-08-17 11:36:45.000000000 +0200
+++ hpgl-distiller-0.9.1/Makefile	2007-08-17 11:36:54.000000000 +0200
@@ -1,7 +1,7 @@
 all: hpgl-distiller
 
 hpgl-distiller: hpgl-distiller.c
-	gcc -Wall hpgl-distiller.c -o hpgl-distiller
+	$(CC) -Wall hpgl-distiller.c -o hpgl-distiller
 
 install: hpgl-distiller
 	cp hpgl-distiller /usr/local/bin
